### PR TITLE
Replace Artemis codename with Snowbridge

### PR DESCRIPTION
**Pull request by vgeddes**
Now that we finally seem to have settled on a name.

The output binaries are now named as follows:
* snowbridge (parachain)
* snowbridge-relay (relayer)

E2E tests all pass

### CODE REVIEW

**Comment by musnit** [comment]
ah awesome thanks

**Review by musnit — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Polkadot Ethereum Bridge
+# Snowbridge
 
-Components for a Polkadot Ethereum Bridge
+A trust-minimized bridge between Polkadot and Ethereum.
```
Comment (philipstanislaus) [inline README.md:3]: Can we say „trustless“ instead? 

I know that there is always smart contract risk and rising the consensus protocols and light client protocols involved, but even if not technically correct, the term “trustless” seems to have become the accepted way to describe that.

File: parachain/runtime/snowbridge/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "snowbridge-runtime"
-description = "Artemis Snowbridge Runtime"
+description = "Snowbridge Snowbridge Runtime"
```
Comment (philipstanislaus) [inline parachain/runtime/snowbridge/Cargo.toml:3]: Dupe

Comment (philipstanislaus) [reply parachain/runtime/snowbridge/Cargo.toml:3]: Might be correct though? Can we rename the runtime/testnet?

**Review by philipstanislaus — COMMENTED** [review]
Thanks!

**Review by philipstanislaus — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (vgeddes) [reply README.md]: sure, fixed in latest commit.

File: parachain/runtime/snowbridge/Cargo.toml (hunk 2 above)
Comment (vgeddes) [reply parachain/runtime/snowbridge/Cargo.toml:3]: I still need to rename the runtime pallets appropriately, so will fix the dupe along with that change.

**Pull request merged**